tests/integration/test_metrics.py

```diff
@@ -739,7 +739,7 @@ def test_transaction_metrics_count_per_root_project(
 
     event, _ = transactions_consumer.get_event()
 
-    metrics_by_project = metrics_by_name_group_by_project(metrics_consumer, timeout=2)
+    metrics_by_project = metrics_by_name_group_by_project(metrics_consumer)
```

Comment:
This halves the timeout from 2 second to 1 second. Can you explain how this improves stability? We usually had to increase the timeout rather than decrease it to avoid flakes.

Reply:
Agreed. FWIW, I haven't heard any complaints about this test flaking in the past two days, so we can also just close this PR and only change something when the test starts flaking again.

Reply:
Agreed let's close it.